tests/test_app/TestApp/Model/Table/GreedyCommentsTable.php

```diff
@@ -27,8 +27,9 @@ public function initialize(array $config): void
      *
      * @param string $type Find type
      * @param array<string, mixed> $options find options
+     * @param mixed ...$args Finder arguments
      */
-    public function find(string $type = 'all', array $options = []): Query
+    public function find(string $type = 'all', array $options = [], ...$args): Query
```

Comment:
I don't know if overriding `find()` is actually supported. If so, this has to be cake 5 only.

Reply:
We need to maintain BC for the public method's signature. It's possible that someone has overridden this method.

Reply:
Ok, cake 5 it is.

Reply:
Users will most certainly have overridden find.